CONTRIBUTORS.txt

```diff
@@ -91,6 +91,7 @@ Eugene Chernyshov
 Eugene Naydenov
 Eugene Tolmachev
 Evert Lammerts
+Fan Yu
```

Comment:
Plz clean-up all unrelated changes.

Reply:
Should I squash all the changes ?

Reply:
Up to you. We'll use "Merge and squash" strategy on this PR anyway :)